src/components/spatial/Spatial.js

```diff
@@ -263,13 +324,18 @@ export default class Spatial extends AbstractSelectableComponent {
     if (images && this.images.length === 0) {
       this.images = Object.entries(images);
     }
-
+    if (raster && this.raster.length === 0) {
```

Comment:
Add explanation?

Reply:
Sorry, could you mention what the explanation is for?